lib/index.js

```diff
@@ -24,7 +24,12 @@ const getEnabledEngines = (configFilePath) => {
 const badCommands = new Set();
 
 const startMeasure = (baseName) => {
-  performance.mark(`${baseName}-start`);
+  const startMark = `${baseName}-start`;
+  // clear start mark from previous execution for the same file
```

Comment:
Good call btw, I think that fixes an odd bug I ran into rarely while testing this.